tests/installation/bootloader.pm

```diff
@@ -210,7 +210,16 @@ sub run() {
 
     my $args = "";
     if ( get_var("AUTOYAST") ) {
-        $args .= " netconfig=dhcp,all";
+        if (get_var("VERSION")=~/^\D*10\D*/ ) { # sle10 does not have netconfig,ifcfg
+            $args .= " netsetup=dhcp,all";
+        }
+        elsif (get_var("VERSION")=~/^\D*11\D*/ ) { # sle11 does not have ifcfg
+            #$args .= " netconfig=dhcp,all";
+            $args .= " netsetup=dhcp,all";  #hotfix, netconfig seems not to work 
```

Comment:
Only that netconfig=dhcp,all does not work on SLES11. Which is the reason for the hotfix commit (up until merge, our autoyast tests were all using netsetup), and probably why all the autoyast tests have been failing on openqa.suse.de